src/assets/database.sql

```diff
@@ -366,6 +366,49 @@ INSERT INTO `commands` (`name`, `description`, `examples`, `tool`, `type`, `comm
 ('Zerocopy', '(client only) Use a "zero copy" method of sending data', 'iperf3 --zerocopy', 19, 'checkbox', '--zerocopy', NULL),
 ('Dont fragment', '(client only) Set the IPv4 Dont Fragment (DF) bit on outgoing packets', 'iperf3', 19, 'checkbox', '--dont-fragment', NULL);
 
+--
+-- Populate data for `commands`: dig
+--
+INSERT INTO `commands` (`name`, `description`, `examples`, `tool`, `type`, `command`, `example`, `sudo`, `category`) VALUES 
+('IPv4 only', 'Use IPv4 query transport only', 'dig -4 example.com', 27, 'checkbox', '-4', NULL, NULL, 'QUERY OPTIONS'),
+('IPv6 only', 'Use IPv6 query transport only', 'dig -6 example.com', 27, 'checkbox', '-6', NULL, NULL, 'QUERY OPTIONS'),
+('addr/port (opt)', 'Bind to source address/port', 'dig -b 10.0.0.1#80', 27, 'input', '-b', NULL, NULL, 'QUERY OPTIONS'), 
+('Query Class', 'The most commonly used class is IN (Internet), which is the default for most DNS queries. However, there are other classes like CH (Chaos) and HS (Hesiod) which are less used', 'dig -c HS example.com', 27, 'input', '-c', NULL, NULL, 'QUERY OPTIONS'),
+('Port Number', 'Specifies the port number for the query', 'dig -p 80 example.com', 27, 'input', '-p', NULL, NULL, 'QUERY OPTIONS'),
+('Query Type', 'Specifies the query type', 'dig -t MX example.com', 27, 'input', '-t', NULL, NULL, 'QUERY OPTIONS'),
+('Reverse Lookup', 'Prepare the query name to be used for reverse lookup (in-addr.arpa)', 'dig -x 10.0.0.1', 27, 'checkbox', '-x', NULL, NULL, 'QUERY OPTIONS'),
```

Comment:
This option requires a target, but I understand what you did by just using the main command target. But this way, if a user check any other option it would not execute. For example: user set target as any IP address then check this reverse lookup and short answer checkboxes. This way the command would end up as being `dig -x +short IP`, which can't execute.

I possible workaround would be move this option as being the last one for this tool once Secflood select it from database or process it. A better solution would, someway, prioritize it to be the last option set.

But, yet, it can be simpler just set it as `input` option so user need to actually pass an IP address to it.

However, any other way can work better for you. Please, let me know :)

Reply:
```suggestion
('Reverse Lookup', 'Query a name in reverse lookup format (in-addr.arpa)', 'dig -x 10.0.0.1', 27, 'input', '-x', NULL, NULL, 'QUERY OPTIONS'),
```